tests/test_examples.py

```diff
@@ -1,13 +1,19 @@
 import os
 import subprocess
 import sys
+from glob import glob
 from pathlib import Path
+from pickle import load
```

Comment:
nit: slight preference for qualified import here (`import pickle`), since `load` is a generic-sounding name

Reply:
Agreed -- changed in new files in https://github.com/proteneer/timemachine/pull/718/commits/35bb6b37b036c166881ed38ee0596c04c550b5e0.

Note 1 other usage (in [`fe/model.py`](https://github.com/proteneer/timemachine/blob/c1a64ddfce22e90424586b0b804d8733ae285f70/timemachine/fe/model.py#L2) ) we might want to change in a future PR